eZ/Bundle/EzPublishCoreBundle/Resources/config/services.yml

```diff
@@ -118,3 +121,11 @@ services:
             - [setFragmentPath, [%fragment.path%]]
             - [setSiteAccess, [@ezpublish.siteaccess]]
         abstract: true
+
+    ezpublish.converter.repository:
```

Comment:
I named it `ezpublish.converter` to follow the same principles as the ones present in Symfony (and in examples) thinking it was a kind of naming rule. I can call it `param_converter` if you guys like it better.

Regarding spiting it, I don't mind, It's also up to you guys. What do you think @bdunogier  ?